downloads/swan-lake-release-notes/swan-lake-2201.10.0/RELEASE_NOTE.md

```diff
@@ -71,12 +71,18 @@ To view bug fixes, see the [GitHub milestone for Swan Lake Update 10 (2201.10.0)
 
 #### Language Server
 
+- Introduced the `Convert to configurable` code action to convert a module level variable into a configurable variable.
+- Introduced the `Extract to configurable` code action to extract expressions or function arguments into configurable variables.
+- Introduced the `Add to Config.toml` code action to add configurable variables in to the Config.toml file.
```

Comment:
```suggestion
- Introduced the `Add to Config.toml` code action to add configurable variables into the `Config.toml` file.
```

Reply:
Nipuna's suggestion, but replacing "into" with just "to".